### PR TITLE
ref(*): stepping down as maintainer

### PR DESCRIPTION
**Pull request by eduser25**
It's been an absolute pleasure and a privilege to be able
to contribute to OSM. I won't be having the necessary cycles
to contribute to the project going onwards, I think there's plenty
of better candidates to take on the maintainer's role already.

Best of luck to OSM team,
Eduard

**Description**:

**Testing done**:

**Affected area**:
| Functional Area            |     |
| -------------------------- | --- |
| New Functionality          | [ ] |
| CI System                  | [ ] |
| CLI Tool                   | [ ] |
| Certificate Management     | [ ] |
| Control Plane              | [ ] |
| Demo                       | [ ] |
| Documentation              | [ ] |
| Egress                     | [ ] |
| Ingress                    | [ ] |
| Install                    | [ ] |
| Networking                 | [ ] |
| Observability              | [ ] |
| Performance                | [ ] |
| SMI Policy                 | [ ] |
| Security                   | [ ] |
| Sidecar Injection          | [ ] |
| Tests                      | [ ] |
| Upgrade                    | [ ] |
| Other                      | [ ] |

Please answer the following questions with yes/no.

1. Does this change contain code from or inspired by another project?
    -   Did you notify the maintainers and provide attribution?

2. Is this a breaking change?

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ref: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
-* @shashankram @snehachhabria @nojnhuh @draychev @eduser25
+* @shashankram @snehachhabria @nojnhuh @draychev
 
 # Emeritus maintainers
-# @michelleN
\ No newline at end of file
+# @michelleN
```
Comment (trstringer) [inline CODEOWNERS:6]: You should be added as an emeritus maintainer!

```suggestion
# @michelleN @eduser25
```

**Review by trstringer — CHANGES_REQUESTED** [review]
Thank you for all your contributions!!

File: CODEOWNERS (hunk 1 above)
Comment (eduser25) [reply CODEOWNERS]: thanks Thomas

**Comment by phillipgibson** [comment]
It was a pleasure having you involved @eduser25!!! Thanks for all you efforts.

**Review by nojnhuh — APPROVED** [review]
Thanks for everything @eduser25! It was great to have you on the team.

**Comment by shashankram** [comment]
@eduser25 given the nature of this PR, it seems that your Github account may have been hacked 👀

**Review by trstringer — APPROVED** [review]

**Review by shashankram — APPROVED** [review]

**Pull request merged**